src/types/index.ts

```diff
@@ -996,6 +996,7 @@ export type WaitForTransactionOptions = {
   timeoutSecs?: number;
   checkSuccess?: boolean;
   indexerVersionCheck?: boolean;
+  indexerTimeoutSecs?: number;
```

Comment:
I agree- let's not rely too much on configuration options, because too many will be overwhelming and difficult for people to find. This is more true when the preferred config/option is going to be very consistent (i.e., 95%+ of developers will expect it by default)